Opencensus Metric collection example failing with ClassNotFoundException

Quickstart example that is given in the opencensus repo failing with below exception.

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/api/client/json/jackson2/JacksonFactory
	at io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration.<clinit>(StackdriverStatsConfiguration.java:50)
	at com.example.opencensus.Quickstart.main(Quickstart.java:26)
Caused by: java.lang.ClassNotFoundException: com.google.api.client.json.jackson2.JacksonFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	... 2 more
```

I have just cloned the repo into my machine to ran the example. Did not make changes to either code or pom.xml, still I a getting the aboe exception.

Am I missing something here?

--- a/opencensus/pom.xml
+++ b/opencensus/pom.xml
@@ -39,6 +39,20 @@
     <opencensus.version>0.31.1</opencensus.version>
   </properties>
 
+ <!--  Using libraries-bom to manage versions.
+  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>25.0.0</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <!-- [START monitoring_opencensus_metrics_dependencies] -->
     <dependency>
@@ -56,6 +70,10 @@
       <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
       <version>${opencensus.version}</version>
     </dependency>
+    <dependency>
+			<groupId>com.google.cloud</groupId>
+			<artifactId>google-cloud-core</artifactId>
+		</dependency>
     <!-- [END monitoring_opencensus_metrics_dependencies] -->
   </dependencies>
 

--- a/opencensus/pom.xml
+++ b/opencensus/pom.xml
@@ -46,7 +46,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>25.0.0</version>
+        <version>26.1.3</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
@@ -71,9 +71,9 @@
       <version>${opencensus.version}</version>
     </dependency>
     <dependency>
-			<groupId>com.google.cloud</groupId>
-			<artifactId>google-cloud-core</artifactId>
-		</dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
     <!-- [END monitoring_opencensus_metrics_dependencies] -->
   </dependencies>
 

